### DISCUSSION TITLE
Support for Dart Language Flutter

### DISCUSSION THREAD

**Discussion by ZahidUrRehman13** [rabbitmq/rabbitmq-server#9038] [Questions]
Please if possible provide support for Dart language like other languages because there is no proper guidiance for dart language for flutter

**Comment by ZahidUrRehman13** [comment]
Any Help

**Comment by mkuratczyk** [comment]
I think this is a good candidate for a community contribution. I don't think anyone on the RabbitMQ team has ever used flutter and given we hear this question perhaps once per year (if not less), I think there are higher priorities for us than learning a new language and ecosystem. We would certainly appreciate someone contributing this though.
